Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -282,25 +289,47 @@ def set_file_reputation(hash, trust_level, filename, comment):
             return create_error_entry(str(ex))
 
 
-args = demisto.args()
-if demisto.command() == 'test-module':
-    test()
-    demisto.results('ok')
-    sys.exit(0)
-elif demisto.command() == 'file':
-    results = file(args.get('file'))
-    demisto.results(results)
-    sys.exit(0)
-elif demisto.command() == 'tie-file-references':
-    results = file_references(args.get('file'))
-    demisto.results(results)
-    sys.exit(0)
-elif demisto.command() == 'tie-set-file-reputation':
-    results = set_file_reputation(
-        args.get('file'),
-        args.get('trust_level'),
-        args.get('filename'),
-        args.get('comment')
-    )
-    demisto.results(results)
-    sys.exit(0)
+def main():
+    try:
+        args = demisto.args()
+        if demisto.command() == 'test-module':
+            test()
+            demisto.results('ok')
+            sys.exit(0)
+        elif demisto.command() == 'file':
+            results = file(args.get('file'))
+            demisto.results(results)
+            sys.exit(0)
+        elif demisto.command() == 'tie-file-references':
+            results = file_references(args.get('file'))
+            demisto.results(results)
+            sys.exit(0)
+        elif demisto.command() == 'tie-set-file-reputation':
+            results = set_file_reputation(
+                args.get('file'),
+                args.get('trust_level'),
+                args.get('filename'),
+                args.get('comment')
+            )
+            demisto.results(results)
+            sys.exit(0)
+    except Exception:
+        validate_certificates_format()
+
+
+if __name__ in ['__main__', '__builtin__', 'builtins']:
+    broker_ca_bundle = './brokercerts.crt'
```

Comment:
Move this whole logic into lines: 321-334 into main() function

Reply:
This is out of the main because if it is inside the main then all functions that use it (get_client_login, new validation function...) don't recognize it, as it's not in their scope.